code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -645,16 +648,45 @@
 	description = "Spectrocybin is a hallucinogenic chemical found in a unique strain of fungi. Little research has been conducted into the hallucinogenic properties of spectrocybin, though many spiritual creeds utilise the drug in rituals and claim it allows people to act as mediums between the living and dead."
 	reagent_state = LIQUID
 	color = "#800080"
-	strength = 5
+	strength = 5 // Was pretty strong during testing - liver failure before you hit the spooky messages.
+	overdose = 10
 	taste_description = "acid"
-	metabolism = REM
+	metabolism = REM * 0.5 // Reduced from 0.2u/t to 0.1u/t to prolong the progression from good messages to bad.
 	unaffected_species = IS_DIONA | IS_MACHINE
+	var/datum/modifier/modifier
 
 /datum/reagent/toxin/spectrocybin/affect_blood(var/mob/living/carbon/M, var/removed)
 	..()
-	M.hallucination = max(M.hallucination, 50)
+	if(is_overdosed == FALSE)	
```

Comment:
Why not just check if the dosage is over 10 instead of using a new variable?

Reply:
The variable was a tad superfluous admittedly up until the suggested overhaul rework I just pushed. With overdoses now being based on a mixture of volume and dose, it's a tad bit more helpful and looks neater. I can remove the variable if it's still deemed unneeded though.

Reply:
Please reopen this if it's still a problem.